features/support/env.rb

```diff
@@ -88,18 +88,13 @@
 end
 
 Before do
-  begin
-    # We are caching classes, but need to manually clear references to
-    # the controllers. If they aren't clear, the router stores references
-    ActiveSupport::Dependencies.clear
-
-    # Reload Active Admin
-    ActiveAdmin.unload!
-    ActiveAdmin.load!
-  rescue
-    p $!
-    raise $!
-  end
+  # We are caching classes, but need to manually clear references to
+  # the controllers. If they aren't clear, the router stores references
```

Comment:
Such an awesome coincidence this fixes a problem I've been struggling with past two days.  Should be in ActiveAdminRequestHelpers also.